tests/mpm_explicit_usf_test.cc

```diff
@@ -45,17 +45,17 @@ TEST_CASE("MPM 2D Explicit implementation is checked",
     auto mpm = std::make_unique<mpm::MPMExplicit<Dim>>(std::move(io));
 
     // Initialise materials
-    REQUIRE(mpm->initialise_materials() == true);
+    REQUIRE_NOTHROW(mpm->initialise_materials());
```

Comment:
Does `REQUIRE_NOTHROW` indicate which function (e.g. `initialise_materials()`) failed since we removed the `runtime_error` above?

Reply:
`REQUIRE_NOTHROW` checks this function `initialise_materials()` doesn't fail. If there is a problem with the function, this condition will be false and the unit test will fail. 

To answer your question, since we are throwing a runtime error, I have added the function name in the error message, so you'll know which function failed if a function fails. We are not catching it locally, but is caught at a higher level.